packages/nulp_elite/src/pages/SelectPreference.js

```diff
@@ -162,7 +164,7 @@ const SelectPreference = ({ isOpen, onClose }) => {
     setIsLoading(true);
     setError(null);
 
-    const url = `/learner/user/v5/read/${_userId}?fields=organisations,roles,locations,declarations,externalIds`;
+    const url = `/learner/user/v5/read/${_userId}?fields=${userReadParam}`;
```

Comment:
is there any other way to pass params??

eg:  
    const option = {
      url: `${this.config.urlConFig.URLS.USER.GET_PROFILE}${this.userid}`,
      param: this.config.urlConFig.params.userReadParam
    };
    this.learnerService.getWithHeaders(option).subscribe(
.....